src/sentry/models/scheduledeletion.py

```diff
@@ -19,7 +19,7 @@ def default_date_schedule():
     return timezone.now() + timedelta(days=30)
 
 
-@control_silo_model
+@all_silo_model
```

Comment:
Previously we would have been able to get away with running deletions only in the region silos. However, with organizations and integrations in the control silo this gets awkward again.